CMakeLists.txt

```diff
@@ -145,6 +145,9 @@ pybind11_add_module(_pylibROM
 	bindings/pylibROM/algo/pyParametricDMD.cpp
 	bindings/pylibROM/algo/pyNonuniformDMD.cpp
   bindings/pylibROM/algo/pyAdaptiveDMD.cpp
+  bindings/pylibROM/algo/manifold_interp/pyInterpolator.cpp
```

Comment:
Not really an issue in this PR, but is there a reason why the indents within the same section in the CMakeLists.txt are not made uniform?

Reply:
I'm not sure if there is a particular reason why the indentation is not consistent in this section. I went ahead and fixed the indentation in this part since it is a small change.